src/components/BasicAuth/BasicAuthFlow/LandingContent.tsx

```diff
@@ -32,6 +35,8 @@ export function LandingContent({
   const handleUsernameChange = (event: React.FormEvent<HTMLInputElement>) => setUsername(event.currentTarget.value);
   const handlePasswordChange = (event: React.FormEvent<HTMLInputElement>) => setPassword(event.currentTarget.value);
 
+  const providerName = providerInfo.displayName ?? capitalize(provider);
+  const docsURL = providerInfo.apiKeyOpts?.docsURL;
```

Comment:
this should be `providerInfo.basicOpts?.docsURL`

Reply:
oops, good catch